Moli/Core/Screens/EditProfile/SocialFieldView.swift

```diff
@@ -0,0 +1,36 @@
+import SwiftUI
+
+struct SocialFieldView: View {
+    var label: String
+    var imageName: String
+
+    var body: some View {
+        RoundedRectangle(cornerRadius: 8, style: .continuous)
+            .fill(Color.gray100)
+            .frame(height: 48)
+            .overlay {
+                HStack {
+                    Text(label)
+                        .font(.labelSmall)
+                        .foregroundColor(.black)
+                    Spacer()
+                    Image(imageName)
+                        .resizable()
+                        .scaledToFit()
+                        .foregroundColor(.black)
+                        .frame(width: 20, height: 20)
```

Comment:
❓ Känns som att i designen så är Instagram ikonen mycket större? Vet inte om det här kommer göra en stor skillnad, men såg att det var 24x24 i figma
```suggestion
                        .frame(width: 24, height: 24)
```